### ISSUE TITLE
isRunningSchedulerWorker returns false positive when PID is stale

### ISSUE THREAD

**Issue by pwhelan** [wa0x6e/ResqueStatus#8]
The ResqueStatus::isRunningSchedulerWorker method will return true if the PID is no longer linked to a running process.

**Comment by chrisinmtown** [comment]
I have this same problem - attempting to restart a scheduler worker does not work if Redis has stale state about a running process.  I solved it by calling `cake CakeResque.CakeResque stop --all` which seems to clean up that saved state.  In a pinch you can call `cake CakeResque.CakeResque reset`. 

But these are just workarounds.  The right way to fix this is by extending ResqueStatus to discover if the state is stale. Unfortunately I see that this issue just passed its 5-year birthday, which suggests that github repo ResqueStatus is more or less dead.  Is this repo truly abandoned?

**Comment by wa0x6e** [comment]
Yes, unfortunately, it's not maintained anymore, and does not support Cake 3+

**Comment by chrisinmtown** [comment]
:(
